### PR TITLE
Basic support for web browsers

### PR DESCRIPTION
**Pull request by aut0**

### CODE REVIEW

**Comment by andersruneson** [comment]
Did you forget to add weppy.js?

File: server.py [hunk 1]
```diff
@@ -230,17 +231,34 @@ def do_GET(self, body=True):
...
-                    fs = os.fstat(f.fileno())
-                    self.send_header("Content-Length", str(fs.st_size))
-                    self.end_headers()
-                    if body:
-                        shutil.copyfileobj(f, self.wfile)
+		    try:
+		        user_agent = self.headers['user-agent']
+		    except:
+			user_agent = 'Initialize with something'
+		    if not user_agent.startswith('Dalvik'):
```
Comment (SamWhited) [inline server.py]: This should probably be a "contains"... also UA detection makes me cry :'( (though I know we kind of have to in thise case).

**Comment by SamWhited** [comment]
This entire thing feels like it's going to break things and go against user expectation to me.

**Comment by aut0** [comment]
Wait, I forgot to branch as well.

**Pull request closed without merging**